### PR TITLE
Campaign View Component

### PR DESCRIPTION
**Pull request by CharlieLe3**
- Changed tables to have a view button (temp)
- connected campaign so that when you view them they will pass the current campaign clicked

### CODE REVIEW

File: server/config.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 TEAM_NAME = os.environ.get('TEAM_NAME')
 DB_USER = os.environ.get('DB_USER')
 DB_PASSWORD = os.environ.get('DB_PASSWORD')
-SQLALCHEMY_DATABASE_URI = f'postgresql://{DB_USER}:{DB_PASSWORD}@localhost/crm_test'
+SQLALCHEMY_DATABASE_URI = f'postgresql://postgres:123456789@localhost/crm_test'
```
Comment (kimcarey) [inline server/config.py]: You are hardcoding this

**Review by kimcarey — CHANGES_REQUESTED** [review]

File: server/config.py (hunk 1 above)
Comment (CharlieLe3) [reply server/config.py]: yeah sorry was just to test. I didn't even mean to push this

File: client/src/pages/CampaignShow.jsx [hunk 2]
```diff
@@ -11,28 +11,37 @@ import TableContainer from "@material-ui/core/TableContainer";
...
-  }
+      <ListItem key={steps.length + 1} className={classes.listItem}>
+        <CampaignStep key={steps.length + 1} />
+      </ListItem>
+    );
+    addStep(steps.concat(newItem));
+    console.log(steps);
+  };
+
+  console.log(currentCampaign);
```
Comment (kimcarey) [inline client/src/pages/CampaignShow.jsx]: do you want this console.log?

File: client/src/pages/CampaignShow.jsx [hunk 3]
```diff
@@ -11,28 +11,37 @@ import TableContainer from "@material-ui/core/TableContainer";
...
-    )
-    addStep(steps.concat(newItem))
-    console.log(steps)
-  }
+      <ListItem key={steps.length + 1} className={classes.listItem}>
+        <CampaignStep key={steps.length + 1} />
+      </ListItem>
+    );
+    addStep(steps.concat(newItem));
+    console.log(steps);
```
Comment (kimcarey) [inline client/src/pages/CampaignShow.jsx]: do you want this console.log?

**Review by kimcarey — APPROVED** [review]

**Pull request merged**